README.md

```diff
@@ -46,6 +50,7 @@ For instructions on installing Scope on [Kubernetes](https://www.weave.works/doc
 If you have any questions about, feedback for or problems with Scope:
 
 - Read [the Weave Scope docs](https://www.weave.works/docs/scope/latest/introducing/).
+- Learn how to [write custom Weave Scope plugins](https://www.weave.works/documentation/scope-latest-plugins/).
```

Comment:
Please remove this line. I don't think that people in need of help would generally want learn how to create a plugin.